quipucords/tests/api/credential/test_credential.py

```diff
@@ -18,14 +16,23 @@
 from tests.factories import CredentialFactory, SourceFactory
 from tests.mixins import LoggedUserMixin
 
+ACCEPT_JSON_HEADER = {"Accept": "application/json"}
 
-class CredentialTest(LoggedUserMixin, TestCase):
+
+def alt_cred_type(cred_type):
+    """Given a credential type, return an alternate random one."""
+    cred_types = DataSources.values.copy()
+    cred_types.remove(cred_type)
+    return random.choice(cred_types)
+
+
+@pytest.mark.django_db
+class TestCredential(LoggedUserMixin):
```

Comment:
I'm almost sure we don't need `LoggedUserMixin` anymore. I created this mixin for unittest based tests.